docs/pages/push-notifications/what-you-need-to-know.mdx

```diff
@@ -95,6 +95,8 @@ The rule of thumb is to prefer a regular Notification Message if you don't requi
 
 Android has a concept of [Data Messages](https://firebase.google.com/docs/cloud-messaging/concept-options#data_messages). iOS does not have exactly the same concept, but a close equivalent is what we call [Headless Background Notifications](#headless-background-notifications).
 
+You may also come across the term "silent notification" which is a yet another name for notifications that don't present anything to the user - we describe these as [Headless Background Notifications](#headless-background-notifications).
```

Comment:
```suggestion
You may also come across the term "silent notification", which is yet another name for notifications that don't present anything to the user &mdash; we describe these as [Headless Background Notifications](#headless-background-notifications).
```

As per our writing style guide, to split sentences, let's use mdash: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#use-mdash